xml/deploy_cephadm.xml

```diff
@@ -527,6 +527,23 @@ o- ssh ................................................... [Key Pair set]
   o- private_key ...... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
   o- public_key ....... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
 </screen>
+   <note>
+     <para>
+       The resulting exported JSON configuration includes a private SSH key.
+       Take care when dumping this file to ensure that it can not be accessed
+       by others.
+     </para>
+     <para>
+       To ensure this does not occur, you can take an extra precaution by
+       using <command>umask</command>:
+     </para>
+<screen>&prompt.cephuser;umask 077
+&prompt.cephuser;ceph-salt export > cluster.json</screen>
+     <para>
+       The <command>umask</command> change avoids accidental granting of
```

Comment:
"Granting of non-root access" sounds weird. How about: "... avoids that the file's content can be read by non-root users."

Reply:
Yeah it is repeating. I think it would be fine if we just removed this - thoughts?

Reply:
Sounds good to me!